src/identity/components/Address.tsx

```diff
@@ -36,7 +47,7 @@ export function Address({
           setTimeout(() => setCopyText('Copy'), 2000);
         })
         .catch((err) => {
-          console.error('Failed to copy address: ', err);
+          console.error('Failed to copy: ', err);
         });
     }
   };
```

Comment:
i know this is unchanged, but making a span behave like a button instead of using a button will never be as good, maybe we can revisit this